test/integration/scheduler/taint_test.go

```diff
@@ -571,3 +575,228 @@ func TestTaintNodeByCondition(t *testing.T) {
 		})
 	}
 }
+
+// TestTaintBasedEvictions tests related cases for the TaintBasedEvictions feature
+func TestTaintBasedEvictions(t *testing.T) {
```

Comment:
Don't we need to remove e2e test `taint_based_evictions.go` according to the PR title?
The original PR (https://github.com/kubernetes/kubernetes/pull/80817) also removed the e2e test.

Reply:
done, looks like that got lost in some of my rebasing